From f9778a6e377f17419d11cce8ba194966bc9c5b58 Mon Sep 17 00:00:00 2001
From: Michael S. Tsirkin <mst@redhat.com>
Date: Wed, 7 Apr 2010 14:25:21 -0300
Subject: [PATCH 08/12] virtio-pci: fix coding style

RH-Author: Michael S. Tsirkin <mst@redhat.com>
Message-id: <fa07098f056638ac39ca6b17257e219bcee8cc55.1270649651.git.mst@redhat.com>
Patchwork-id: 8427
O-Subject: [PATCH RHEL6.0 5/9] virtio-pci: fix coding style
Bugzilla: 580109
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>

Fix coding style violations and gratituos diff from upstream
in virtio-pci.

Cherry-picked from commit ade80dc84527ae7418e9fcaf33e09574da0d2b29.

Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
---
 hw/virtio-pci.c |    7 ++++---
 1 files changed, 4 insertions(+), 3 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/virtio-pci.c |    7 ++++---
 1 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/hw/virtio-pci.c b/hw/virtio-pci.c
index f557caf..f55483d 100644
--- a/hw/virtio-pci.c
+++ b/hw/virtio-pci.c
@@ -431,8 +431,9 @@ static int virtio_pci_set_guest_notifier(void *opaque, int n, bool assign)
 
     if (assign) {
         int r = event_notifier_init(notifier, 0);
-	if (r < 0)
-		return r;
+        if (r < 0) {
+            return r;
+        }
         qemu_set_fd_handler(event_notifier_get_fd(notifier),
                             virtio_pci_guest_notifier_read, NULL, vq);
         msix_set_mask_notifier(&proxy->pci_dev,
@@ -679,9 +680,9 @@ static PCIDeviceInfo virtio_info[] = {
             DEFINE_PROP_UINT32("vectors", VirtIOPCIProxy, nvectors,
                                DEV_NVECTORS_UNSPECIFIED),
             DEFINE_PROP_HEX32("class", VirtIOPCIProxy, class_code, 0),
+            DEFINE_VIRTIO_COMMON_FEATURES(VirtIOPCIProxy, host_features),
             DEFINE_PROP_UINT32("max_ports", VirtIOPCIProxy, max_virtserial_ports,
                                31),
-            DEFINE_VIRTIO_COMMON_FEATURES(VirtIOPCIProxy, host_features),
             DEFINE_PROP_END_OF_LIST(),
         },
         .qdev.reset = virtio_pci_reset,
-- 
1.7.0.3

